src/main/java/com/impactupgrade/nucleus/service/segment/StripeDataSyncService.java

```diff
@@ -42,7 +54,51 @@ public void syncContacts(Calendar updatedAfter) throws Exception {
   public void syncTransactions(Calendar updatedAfter) throws Exception {
     List<CrmDonation> crmDonations = env.primaryCrmService().getDonations(updatedAfter);
     for (CrmDonation crmDonation : crmDonations) {
-      //TODO: bulk update in Stripe
+      //TODO: one by one update in Stripe
     }
   }
+
+  protected Customer updateOrCreateCustomer(CrmContact crmContact) throws Exception {
+    Optional<Customer> existingCustomer = getCustomer(crmContact);
+    if (existingCustomer.isPresent()) {
+      // Update
+      CustomerUpdateParams customerUpdateParams = CustomerUpdateParams.builder()
+          .setName(crmContact.getFullName())
+          .setEmail(crmContact.email.toLowerCase(Locale.ROOT))
+          .setAddress(toUpdateAddress(crmContact.mailingAddress))
```

Comment:
@VSydor Use `crmContact.account.billingAddress` instead

Reply:
@nqpatterson Anything else DR US wanted updated? This will do name + physical address, currently.

Reply:
Are email and phone possible?

Reply:
@nqpatterson Sorry, didn't see this comment! V has it set up for phone now, but emails will be really tough. I can put together some info for DR US/AU explaining it.